mw-webapp/src/component/table/columns.tsx

```diff
@@ -43,6 +58,9 @@ const renderStringCell = ({text, key, isDone}: {text:string; key:string; isDone?
   );
 };
 
+/**
+ * Render cell with boolean value
+ */
```

Comment:
Not consistent name and comment, check everywhere

Reply:
Changed